connector/dialect.tdd

```diff
@@ -770,63 +413,88 @@
       <argument type='int' />
       <argument type='int' />
     </function>
+    <function group='operator' name='+' return-type='real'>
+      <formula>(%1 + %2)</formula>
+      <argument type='int' />
+      <argument type='real' />
+    </function>
+    <function group='operator' name='+' return-type='real'>
+      <formula>(%1 + %2)</formula>
+      <argument type='real' />
+      <argument type='int' />
+    </function>
     <function group='operator' name='+' return-type='str'>
```

Comment:
From a usability perspective, this concerns me. In the core MongoSQL syntax, `+` has one meaning: arithmetic addition. It is invalid to use it for string concatenation. It looks like here we're saying Tableau users will be able to do string concatenation via the `+` operator. If users get used to that and then move to other tools (or possibly even just write custom SQL in tableau) they may get confused why suddenly `+` doesn't work for string concatenation. We should discuss as a team if we want to expose this operator this way.

Reply:
Sounds good, I've removed it.